### ISSUE TITLE
Improve camera settings display

### ISSUE THREAD

**Issue by Maxou44** [brick-a-brack/eagle-animation#100]
- Allow the use of arrow to navigate between tabs
- Use a fixed height and a scroll instead of a dynamic height
- Show disabled values instead of hide them when it's not possible (like focus when the AF is enabled)

**Comment by Maxou44** [comment]
Add the ability to reset each setting individualy

**Comment by Maxou44** [comment]
After some discussions with @4clem4, this is our ideas:
- Split all the settings into different tabs
- Rework sliders to make them more precise and easier to understand by using colors and icons
- Fix the height of the camera settings popup
- Reduce the opacity of the camera settings popup when we are editing a value
- Add the ability to reset each setting individualy (when supported)

**Comment by Maxou44** [comment]
Logitech G Hub : 

<img width="294" height="721" alt="Image" src="https://github.com/user-attachments/assets/85c42c06-3000-46ba-a54f-a7ed87b6f811" />

<img width="256" height="661" alt="Image" src="https://github.com/user-attachments/assets/c3632674-23e4-456e-9c60-805d45cfcf70" />

<img width="302" height="668" alt="Image" src="https://github.com/user-attachments/assets/73660384-9f96-4375-9488-727291104256" />

<img width="293" height="700" alt="Image" src="https://github.com/user-attachments/assets/493c56d7-eb3f-4104-9099-f97fabec4882" />

**Comment by FinixFighter** [comment]
I also think that while adjusting settings the preview should remain with the same brightness, otherwise the user can't really evaluate the changes he's making; he/she always have to close the settings to look at the preview in full brightness and then go back to the setting to continue adjusting.
Preview brightness, quality etc. should always stay the same.

**Comment by Maxou44** [comment]
> I also think that while adjusting settings the preview should remain with the same brightness, otherwise the user can't really evaluate the changes he's making; he/she always have to close the settings to look at the preview in full brightness and then go back to the setting to continue adjusting. Preview brightness, quality etc. should always stay the same.

We're not really able to do this, as DSLR cameras provide a live stream that differs from the actual captured frame. Some SDKs try to fake it so that it looks closer to the final photo, but this depends on the brand's SDK and on the device itself, and we can't realistically simulate that manually for every model.

This is a fundamental limitation of tethered shooting rather than something specific to Eagle Animation, you'll run into the same behaviour with Stop Motion Studio or Dragonframe. The usual workflow is to set up a test scene and take a few sample shots to configure your camera settings properly before starting.

It's different with webcams, since the live view and the captured frame come from the same stream.

**Comment by Maxou44** [comment]
Maybe I misunderstood, but is the issue the overlay of the settings panel, or the differences between the live view and the captured frames?

**Comment by FinixFighter** [comment]
> Maybe I misunderstood, but is the issue the overlay of the settings panel, or the differences between the live view and the captured frames?

I mean the difference between the live view when the settings panel is closed and the live view when the settings panel is open. 
When the settings panel is open the portion of the screen containing the live view turns darker, so it's almost like making adjustments without being able to see the changes live.
In these two examples you can see the difference between the brithness when settings panel is closed and when setting panel is open.
<img width="1918" height="1075" alt="Image" src="https://github.com/user-attachments/assets/7edd4d47-2371-4bab-aed8-b92645f2a447" />
<img width="1937" height="1097" alt="Image" src="https://github.com/user-attachments/assets/cdd50c63-b919-4165-b606-aade4d32187b" />
The second one is darker (and even partially hidden but that's another issue, becase if I have to focus on something that's placed at the bottom of the view then it will be kind of difficult).

Captured frames are ok!

**Comment by Maxou44** [comment]
Thanks for the update 👌
